challenges/3-area-trapezio/python/jayron88/area.py

```diff
@@ -0,0 +1,9 @@
+while True:
```

Comment:
Cara legal, sua lógica funcionou. Mas só um detalhe... Como eu paro seu código, minha sugestão é crie uma condição de parada desse while, pois não é legal termos que apelar para o velho CRTL + C.

Reply:
O que sugere como condição? quantidade de erros? escolha do usuario?

Reply:
Quando o resultado for valido, já pode ter uma condição de parada em seguida